gollm/gemini.go

```diff
@@ -130,15 +122,15 @@ func (c *GeminiClient) GenerateCompletion(ctx context.Context, request *Completi
 }
 
 // StartChat starts a new chat with the model.
-func (c *GeminiClient) StartChat(systemPrompt string) Chat {
+func (c *GeminiClient) StartChat(systemPrompt string, model string) Chat {
```

Comment:
So I was thinking that the model might go into the LLM_CLIENT URI, but let's see how this feels from a caller perspective!  I hadn't anticipated the idea of switching models mid-chat when thinking about a URI.